tests/integration_tests/test_llm.py

```diff
@@ -1254,6 +1255,39 @@ def test_llm_encoding(llm_encoder_config, adapter, quantization, tmpdir):
     model.train(dataset=dataset_path, output_directory=str(tmpdir))
 
 
+def test_llm_batch_size_tuning():
+    dataset = pd.DataFrame({"instruction": ["a"] * 100, "output": ["a"] * 100})
+    config = yaml.safe_load(
+        """
+    model_type: llm
+    input_features:
+        - name: instruction
+          type: text
+    output_features:
+        - name: output
+          type: text
+    prompt:
+        template: >-
+            {instruction}
```

Comment:
@Infernaught Seeing the same test twice? on line 1258 and line 1348

Reply:
I think pre-commit is also complaining about this

Reply:
Interesting. Probably an issue with merging?